### PR TITLE
New Futarchy Programs all V0.3

### PR DESCRIPTION
**Pull request by LukasDeco**
- **feat: SDK with new amm**
- **feat: new IDLs and removal of v1**

### CODE REVIEW

File: lib/client/rpc/dao.ts [hunk 1]
```diff
@@ -118,47 +118,72 @@ export class FutarchyRPCDaoClient implements FutarchyDaoClient {
...
-  async getMinLpToProvide(daoAggregate: DaoAggregate): Promise<{ base: BN, quote: BN } | undefined> {
-    const autocrat = new Program<AutocratV1>(AUTOCRAT_V1_IDL, "5scUH1qY87Awh5D8bQkbP2uv9BaXJXiMgrqRBw1K2vVv", this.rpcProvider)
-    const currentDao = daoAggregate.daos.filter((dao) => dao.protocol.deploymentVersion === "V1").slice(-1)[0]
-    if (!currentDao) return
+  async getMinLpToProvide(
+    daoAggregate: DaoAggregate
+  ): Promise<{ base: BN; quote: BN } | undefined> {
+    const autocrat = new Program<Autocrat>(
+      AUTOCRAT_V0_3_IDL,
+      "5scUH1qY87Awh5D8bQkbP2uv9BaXJXiMgrqRBw1K2vVv",
```
Comment (R-K-H) [inline lib/client/rpc/dao.ts:136]: `AMM5G2nxuKUwCLRYTW7qqEwuoqCtNSjtbipwEmm2g8bH` is the new program

File: lib/client/rpc/proposals/finalizeProposal.ts [hunk 2]
```diff
@@ -229,7 +225,7 @@ export class FinalizeProposalClient implements FinalizeProposal {
     switch (proposal.protocol.deploymentVersion) {
       case "V0.2":
         return await this.finalizeProposalv02(proposal);
-      case "V1":
+      case "V0.3":
         return await this.finalizeProposalv1(proposal);
```
Comment (R-K-H) [inline lib/client/rpc/proposals/finalizeProposal.ts:229]: v1?

File: lib/constants/index.ts [hunk 3]
```diff
@@ -33,40 +32,34 @@ export const QUOTE_LOTS = 0.0001;
...
-  },
   {
     label: "V0.3",
-    programId: new PublicKey("FuTPR6ScKMPHtZFwacq9qrtf9VjscawNEFTb2wSYr1gY"),
+    programId: new PublicKey("autoQP9RmUNkzzKRXsMkWicDVZ3h29vvyMDcAYjCxxg"),
     idl: AUTOCRAT_V0_3_IDL as AutocratProgram,
-    dao: "multidao",
+    dao: "multidao"
   },
   {
```
Comment (R-K-H) [inline lib/constants/index.ts:41]: Nix this...

File: lib/constants/twap.ts [hunk 4]
```diff
@@ -25,25 +25,21 @@ export const autocratVersionToTwapMap: Record<
...
     programId: OPENBOOK_TWAP_PROGRAM_IDV0_2,
-    idl: OPENBOOK_TWAP_IDLV0_2,
+    idl: OPENBOOK_TWAP_IDLV0_2
   },
   "V0.3": {
-    programId: OPENBOOK_TWAP_PROGRAM_IDV0_2,
-    idl: OPENBOOK_TWAP_IDLV0_2,
-  },
-  "V1": {
     programId: OPENBOOK_TWAP_PROGRAM_IDV0_2,
```
Comment (R-K-H) [inline lib/constants/twap.ts:39]: v0_3

File: lib/client/rpc/proposals/finalizeProposal.ts (hunk 2 above)
Comment (LukasDeco) [reply lib/client/rpc/proposals/finalizeProposal.ts]: fixed the name of the method

File: lib/client/rpc/dao.ts (hunk 1 above)
Comment (LukasDeco) [reply lib/client/rpc/dao.ts]: added new autocrat

**Pull request merged**